ethportal-api/src/types/portal.rs

```diff
@@ -53,7 +53,7 @@ pub struct TraceGossipInfo {
     pub transferred: Vec<String>,
 }
 
-/// Response for FindContent & RecursiveFindContent endpoints
+/// Response for FindContent & GetContent endpoints
```

Comment:
While it's true that this is the response type, RecursiveFindContent would only return `Content` variant (this is also according to spec). Should we create separate type for it?

I also believe that `ConnectionId` variant is never used (not even for `FindContent`). Should we remove it?

While it's not strictly related to this pr (renaming endpoint), I think that this is something that we should improve. If you feel like it, maybe do it in this PR, otherwise create an issue for it.

Reply:
I will make a follow up PR to resolve this